sphinx/search/__init__.py

```diff
@@ -224,7 +226,7 @@ class IndexBuilder:
     passed to the `feed` method.
     """
     formats = {
-        'jsdump':   jsdump,
+        'json':     json,
```

Comment:
I noticed sphinxcontrib-serializinghtml package depends on "jsdump" format. So it should be also updated after v5.0 released.